### DISCUSSION TITLE
Vite Dependency Pre-Bundling too large?

### DISCUSSION THREAD

**Discussion by trananhmanh89** [tusen-ai/naive-ui#1669] [Q&A] [answered]
Hi,
I'm new with vite
Today i try to test vite with naive-ui. On dev mode, naive-ui pre bundling is seem very large (~8mb) while i only import a Button.
Is this normal behavior of vite? And any way to decreses size of Dependency Pre-Bundling ?
Thanks

![2021-11-24_140148](https://user-images.githubusercontent.com/20571336/143196019-51ab9733-658f-4501-b918-d8fb0450c616.png)

![image](https://user-images.githubusercontent.com/20571336/143196140-e4fa95fc-7208-4aaf-8f86-fbafc1168ac1.png)

**Comment by 07akioni** [comment, marked as answer]
This is normal, never mind. Prod build would be treeshakable.
